### PR TITLE
Add keyring dependencies

### PR DESCRIPTION
**Pull request by genestack-solomatin**
There is trouble on installing python client on ubuntu.  

Keyring 8.0 have changes that don't see default security backend on ubunty.  https://github.com/jaraco/keyring/issues/117

Simple solution is to install `keyrings.alt` package, but It is possible that FileBackend will be used(base64 encoded pasword in plain text file).

To test your platform you need to install `keyring` and `keyrings.alt` and execute code:

``` python
import keyring.core
print keyring.core._keyring_backend
```

### CODE REVIEW

**Comment by klas-genestack** [comment]
> Simple solution is to install `keyrings.alt` package, but It is possible that FileBackend will be used(base64 encoded pasword in plain text file).

In which cases this would be used?  We could check if it's selected backend and fail/warn.

File: setup.py [hunk 1]
```diff
@@ -13,7 +13,7 @@
     author='Genestack Limited',
     author_email='',
     description='Genestack Python Client Library',
-    install_requires=['keyring', 'requests'],
+    install_requires=['keyring', 'requests', 'keyrings.alt'],
```
Comment (klas-genestack) [inline setup.py]: How about
```python
install_requires=['keyring', 'requests'] + ['keyrings.alt'] if sys.platform.startswith('linux') else [],
```

**Comment by genestack-solomatin** [comment]
On ubuntu
```
>>> import keyring.core
>>> print keyring.core._keyring_backend
<keyring.backends.file.EncryptedKeyring object at 0x7f8d24c44750>
```

**Comment by klas-genestack** [comment]
Yeah, I know, but it doesn't answer my question.
You say that insecure backend could be used, so I ask "when" and "should we guard against it".

**Comment by klas-genestack** [comment]
[My question](https://github.com/genestack/python-client/pull/155#issuecomment-233904989) still stands.

**Comment by genestack-solomatin** [comment]
Not actual.

**Pull request closed without merging**